lxd/cgroup/abstraction.go

```diff
@@ -1046,6 +1047,11 @@ func (cg *CGroup) GetIOStats() (map[string]*IOStats, error) {
 					continue
 				}
 
+				// Skip irrelevant stats related to direct IO (dbytes= and dios=).
```

Comment:
this wasnt skipped before right? So the will the metrics values change as a result of this, and are you saying they were wrong before?

Reply:
It's always been skipped but just a bit further down in the function when comparing the key names:

```
                switch statName {
                case "rbytes":
                    ioStats.ReadBytes = statValue
                case "wbytes":
                    ioStats.WrittenBytes = statValue
                case "rios":
                    ioStats.ReadsCompleted = statValue
                case "wios":
                    ioStats.WritesCompleted = statValue
                }
```

Reply:
ok thanks, this wasnt clear at all.